cmd/importer/importer.go

```diff
@@ -27,21 +26,13 @@ func main() {
 	ep := ParseEnvVar(common.IMPORTER_ENDPOINT, false)
 	acc := ParseEnvVar(common.IMPORTER_ACCESS_KEY_ID, false)
 	sec := ParseEnvVar(common.IMPORTER_SECRET_KEY, false)
-	importInfo, err := NewImportInfo(ep, acc, sec)
-	if err != nil {
-		glog.Fatalf("main: unable to get env variables: %v\n", err)
-	}
-	importInfo.Url, err = url.Parse(importInfo.Endpoint)
-	if err != nil {
-		glog.Fatalf("main(): \n")
-	}
-	dataReader, err := NewDataReader(importInfo)
+	dataStream, err := NewDataStreamFactory(ep, acc, sec).NewDataStream()
 	if err != nil {
-		glog.Fatalf("main: unable to create data reader: %v\n", err)
+		glog.Fatalf("main: error getting data stream: %v", err)
 	}
-	defer dataReader.Close()
-	glog.Infof("Beginning import from %s\n", importInfo.Url.RawPath)
```

Comment:
Do we still want the func name (e.g. _main:_) in err / log msgs? Most of the msgs here include the func name but maybe that's temp for initial debugging?

Reply:
We probably should.  My method up to now was to only do it in errors, but it's wise to mark all log output that way.  Too bad glog doesn't do that for us.  There are _other_ logging packages that will... But I don't think we want to vendor in some random guy's 3rd party package.